public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/UpdateOperatorModal/UpdateOperatorModal.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ palette, spacing }: GrafanaTheme) => ({
+  modalWrapper: css`
+    div[data-qa='modal-body'] {
+      left: 27%;
```

Comment:
Does it render well on different screen sizes?

Reply:
![Screenshot 2021-09-03 at 10 09 19](https://user-images.githubusercontent.com/16366502/131980925-11b01453-8654-48bf-886c-6d5cf742ba08.png)
![Screenshot 2021-09-03 at 10 09 34](https://user-images.githubusercontent.com/16366502/131980930-bd79d5c0-20b0-4818-9693-ab13db71362a.png)
![Screenshot 2021-09-03 at 10 09 50](https://user-images.githubusercontent.com/16366502/131980932-f30f0e6a-6947-4a5e-ac08-bedd9f96754a.png)
![Screenshot 2021-09-03 at 10 09 59](https://user-images.githubusercontent.com/16366502/131980934-c29e6f47-f67b-4ffc-80c8-cbe79fb7aa18.png)